docs/tutorials/nuvlabox/gpu-demo.md

```diff
@@ -1,73 +1,99 @@
-        ports:
-            - "5000:5000"
-        devices:
-            - /dev/nvidiactl
-            - /dev/nvidia-uvm
-            - /dev/nvidia0
-        volumes:
-            - /usr/bin/:/usr/bin/
+  darknet:
+    image: 'franciscomendonca/darknet:1.0'
+    ports:
+      - "8000:8000"
+    volumes: 
+      - '/usr/local/cuda-10.0/:/usr/local/cuda-10.0/'
+      - '/usr/lib/aarch64-linux-gnu:/usr/lib/aarch64-linux-gnu' 
+      - '/usr/local/cuda/lib64:/usr/local/cuda/lib64'
+    devices:
+      - '/dev/nvhost-gpu:/dev/nvhost-gpu'
+      - '/dev/nvhost-as-gpu:/dev/nvhost-as-gpu'
+      - '/dev/nvhost-ctrl:/dev/nvhost-ctrl'
+      - '/dev/nvhost-ctrl-gpu:/dev/nvhost-ctrl-gpu'
+      - '/dev/nvhost-prof-gpu:/dev/nvhost-prof-gpu'
+      - '/dev/nvmap:/dev/nvmap'
+      - '/dev/video0:/dev/video0'
+    environment:
+      - LD_LIBRARY_PATH=:/usr/lib/aarch64-linux-gnu:/usr/lib/aarch64-linux-gnu/tegra:/usr/local/cuda/lib64:/usr/local/cuda/lib64:/root/opencv-3.4.1/build/lib
 ```
 
-As you can see, to be able to use the GPU of the device, through Docker-Compose we need to pass the device path - using the devices part - and
-the folders where the drivers and libraries are located - using the volumes. In this case, we pass the full /usr/bin/ folder, but you can pass
-only the folders you require.
 
-## Deployment
+As you can see, since we can not use `--gpus` in a Compose file (yet), we need to define the different host `devices` to be mapped into the container, alongside a mount for the folders where the GPU drivers and libraries are located, on the host - in this case, we pass the full /usr/bin/ folder, but we can pass only the specific sub-folders we require. 
 
-In Nuvla.io, go to the App Store and find the app called **"GPU Demo”**. Click **“launch”**
-A panel with appear - select your NuvlaBox credentials. 
-As we don't have environmental variables or files, you can click **"Deploy"**
+With this, you can create your own Nuvla app, by following [these steps](http://localhost:4000/nuvla/add-apps).
+ 
+### Deploy the App into the NVIDIA Jetson Nano
+
+In Nuvla.io, go to the App Store and find the app called **"GPU Demo”**. We've pre-created one for your with the instructions from above. 
+
+Click **“launch”** and a deployment modal will appear - select your NuvlaBox credentials. 
+
+As we don't have environmental variables or files for this application, we can simply click **"Deploy"**.
 
 At start, the app will download and build Darknet - this will take a while.
-Afterwards, it will launch the web server, so at this point you can access the web interface:
+Afterwards, it will launch a web server with the video feed, and the app's dynamic URL will be filled in Nuvla. So at this point we can access the web interface:
     
-    http://{IP of your machine}:5000
+    http://{IP of your machine}:5000    
+
+And the interface should look something like this:
+
+
+![gpuDemo](/assets/img/gpu-demo-interface.png){: :center}
```

Comment:
same question as above, about rights to use the image

Reply:
I'll change for the same image as above